syftlib/src/main/java/org/openmined/syft/Syft.kt

```diff
@@ -49,14 +73,12 @@ class Syft(
     }
 
     @ImplicitReflectionSerializer
-    fun send(message: String) {
-        signallingClient.send(
-            "Personal", JsonObject(
-                mapOf(
-                    "data" to JsonPrimitive(message),
-                    "workerId" to JsonPrimitive("1234")
-                )
+    fun send(message: String) = signallingClient.send(
+        "Personal", JsonObject(
```

Comment:
This can be simplified to json { "data" to JsonPrimitive()}

Reply:
Noted. This should go. It's only for demo purposes.